### DISCUSSION TITLE
INFO aliyundrive_webdav: Please scan the qrcode to login in 30 seconds

### DISCUSSION THREAD

**Discussion by qq7171** [messense/aliyundrive-webdav#474] [Q&A]
![image](https://user-images.githubusercontent.com/32334113/176984550-18d659d2-824e-4290-8ebe-d3566e00a56b.png)
昨天阿里云盘用不起了，也更新了1.51，然后就这个提示了  群晖docker环境使用,这在哪里扫描啊? 

今天下午我偿试用openwrt方式  成功了

**Comment by Mechanic-Hwang** [comment]
同样出现了这个问题，可能是因为我高频率操作，或者长时间没有登录？现在还不知道怎么解决。

**Comment by winglight** [comment]
进了容器bash，但是显示的qr code有问题，无法扫描。。。

**Comment by lionnd** [comment]
同问，挠头
